modules/groovy-dsl/src/test/groovy/io/swagger/assert4j/dsl/RestRequestDslTest.groovy

```diff
@@ -52,6 +52,24 @@ class RestRequestDslTest {
     }
 
     @Test
+    void buildsRecipeWithPOSTAndBody() throws Exception {
+        String body = """
+            {
+                "customerId": "some_id",
+                "lineItems": [
+                  { sku: 'ABC-123', quantity: 2},
+                  { sku: 'DEF-456', quantity: 1}
+            }
+            """
+        TestRecipe recipe = recipe {
+            post URI, body
```

Comment:
how do you set the content-type of the body?

Reply:
To do this you need to use the more complex syntax, where you pass in a closure instead of a string. What do you think about setting application/json as the default?

Reply:
Hmm ... Turns out that we didn't have support for setting mediaType in the DSL will add that too.

Reply:
thanks! perhaps we can even deduce the media-type for json and xml - which I'm guessing are the most common formats!?